db/notifications.go

```diff
@@ -125,7 +125,7 @@ func NotificationSendSummary(ctx context.Context, id string, window time.Duratio
 	return earliest.Time, count, err
 }
 
-func GetMatchingNotificationSilencesCount(ctx context.Context, resources models.NotificationSilenceResource) (int64, error) {
+func GetMatchingNotificationSilences(ctx context.Context, resources models.NotificationSilenceResource) ([]models.NotificationSilence, error) {
```

Comment:
I think `	orClauses := ctx.DB().Where("1 = 0")` needs to change where filter is not null